imgoing-frontend/App.tsx

```diff
@@ -3,26 +3,32 @@ import { ThemeProvider } from 'styled-components/native';
 import * as Font from 'expo-font';
 import { Provider } from 'react-redux';
 import { RootSiblingParent } from 'react-native-root-siblings';
-
+import * as SplashScreen from 'expo-splash-screen';
```

Comment:
```suggestion
import SplashScreen from 'expo-splash-screen';
```
 as 제거해주면 좋겠씁니당! 하시면서 위에 Font도 제거해주세요 ㅎㅎ,,

Reply:
이렇게 변경하면 SplashScreen이랑 Font가 적용이 안되네요 ㅠ

Reply:
헉 그렇군요,,